sdk/identity/identity/README.md

```diff
@@ -122,6 +122,9 @@ When you submit a pull request, a CLA-bot will automatically determine whether y
 a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions
 provided by the bot. You will only need to do this once across all repos using our CLA.
 
+## Developer Contributing Guide
+For details on developer inner-loop workflow and contributing guide see the [contributing guide](https://github.com/Azure/azure-sdk-for-js/blob/master/CONTRIBUTING.md)
```

Comment:
Do you mind if I suggest an alternative wording?  Happy to workshop this a bit more to get it perfect

```suggestion
If you'd like to contribute to this library, please read the the [contributing _guide](https://github.com/Azure/azure-sdk-for-js/blob/master/CONTRIBUTING.md) to learn more about how to build and test the code.
```

Reply:
Yes definitely :)

Reply:
done